pkg/client/monitoring/v1/types.go

```diff
@@ -367,6 +367,8 @@ type Endpoint struct {
 	BasicAuth *BasicAuth `json:"basicAuth,omitempty"`
 	// MetricRelabelConfigs to apply to samples before ingestion.
 	MetricRelabelConfigs []*RelabelConfig `json:"metricRelabelings,omitempty"`
+	//Optional ProxyURL
```

Comment:
There is a space missing between `//` and the start of the comment, and this needs a more thorough description of what it does.